lib/WasmReader/BaseWasmReader.h

```diff
@@ -26,6 +26,21 @@ namespace Wasm
         ModuleInfo * m_moduleInfo;
         WasmModule * m_module;
 
+        // TODO: Move this to somewhere more appropriate and possible make m_alloc part of
+        // BaseWasmReader state.
+        char16* CvtUtf8Str(ArenaAllocator& m_alloc, LPUTF8 name, uint32 nameLen)
```

Comment:
generally we pass pointers by value, which makes more explicit at call-site what is happening